ansible/deploy_commcarehq.yml

```diff
@@ -20,3 +20,27 @@
     - role: tmpreaper
       tags: ['commcarehq', 'celery_hq', 'tmpreaper', 'cron']
       tmpreaper_time: '2d'
+
+- name: Pillows Supervisor Config
+  hosts: pillowtop
+  tasks:
+    - include_tasks: roles/commcarehq/tasks/pillowtop.yml
```

Comment:
should probably be `import_tasks`. Then you also wouldn't need to put the `tags` in the tasks files

Reply:
Unless that breaks the `when` checks on the tasks